corehq/apps/reports/templates/reports/reportdata/excel_export_data.html

```diff
@@ -1,388 +0,0 @@
-{% extends report_base|default:"reports/async/bootstrap2/default.html" %}
```

Comment:
seems like this is actually still used though i can't seem to find the De-Identified reports page through the UI